scripts/helmcharts/openreplay/templates/job.yaml

```diff
@@ -103,7 +103,7 @@ spec:
           mountPath: /opt/openreplay
         - name: dbmigrationscript
           mountPath: /opt/migrations/
-      {{- if eq .Values.global.s3.endpoint "http://minio.db.svc.cluster.local:9000" }}
+      {{- if contains "svc.cluster.local" .Values.global.s3.endpoint }}
```

Comment:
I think, search string should contain minio also, because for migration, we're calling minio specific commands.

Reply:
in this case isn't this endpoint either used for S3 or the in cluster minio?